app/ui/ui-emergency/src/main/kotlin/com/hedvig/android/ui/emergency/EmergencyScreen.kt

```diff
@@ -91,14 +99,36 @@ fun EmergencyScreen(emergencyNumber: String?, navigateUp: () -> Unit, modifier:
             ),
             modifier = Modifier.fillMaxWidth(),
           )
-          if (emergencyNumber != null) {
+          if (emergencyUrl != null) {
             Spacer(Modifier.height(24.dp))
+            HedvigContainedSmallButton(
+              text = stringResource(
+                R.string.SUBMIT_CLAIM_GLOBAL_ASSISTANCE_URL_LABEL,
+                emergencyUrl,
+              ),
+              onClick = {
+                openUrl(emergencyUrl)
+              },
+              modifier = Modifier.fillMaxWidth(),
+            )
+          }
+          if (emergencyNumber != null) {
+            val colors = if (emergencyUrl != null) {
```

Comment:
So the color depends on if there are two buttons? So they don't look the same?

Reply:
the way I see it, the top primary button is filled, and the bottom one is transparent, bc we want the member go the top button way (url in this case). But if there is only transparent button (for the call), it looks too bleak alone, and it should become our "primary way" if it is alone. So I did it like this.

Reply:
I agree, it's a good idea